client/components/map.jsx

```diff
@@ -33,10 +32,17 @@ class Map extends Component {
   }
 
   createMarker (lat, lng){
+    
+    const markerIcon = {
+      url: `${window.location.href}airbrb_home.png`,
+      scaledSize: new google.maps.Size(62,62),
```

Comment:
What does this mean
??

Reply:
it's to make the custom marker smaller to that dimension

Reply:
INteresting interesting.